README.md

```diff
@@ -1,27 +1,114 @@
-# WipPlatformFe
+# Setup
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 17.0.8.
+Clone the repository and install packages with: `npm install`
 
-## Development server
+# General rules
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+When writing code we follow the SOLID principles and try to keep the code clean.<br>
+More on SOLID: https://wearecommunity.io/communities/epam-poland/articles/1190<br>
+More on how to keep the code clean: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29
 
-## Code scaffolding
+# Project structure
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+We use the following project structure:
+![image](https://github.com/WIP-Work-In-Progress/wip-platform-fe/assets/143722910/cce925b8-3498-49e0-9eba-8f707ba82c34)<br>
+In the **shared** directory, we follow the SCAM pattern.<br>
+More on the SCAM pattern: https://angular-training-guide.rangle.io/modules/module-scam-pattern<br>
+More on the main parts of an angular application: https://atejszerska.pl/blog/1/details
 
-## Build
+# Designs
```

Comment:
You use "-ing" form in most other headers. I think it would be better to put "designing" here instead of "designs" for consistency

Reply:
I wrote 'Designs' instead of 'Designing' here, because the paragraph contains important information not only for the people wanting to contribute their designs, but also for people, who want to know where we get designs from/what designs we use. I think the current form of the header is more general whereas  "Designing" could be a bit misleading for the second group of people i mentioned

Reply:
I see your point. In this case, we can leave it